elliottlib/cli/find_bugs_cli.py

```diff
@@ -21,20 +21,27 @@
 @use_default_advisory_option
 @click.option("--mode",
               required=True,
-              type=click.Choice(['list', 'sweep', 'diff', 'qe']),
+              type=click.Choice(['list', 'sweep', 'diff', 'qe', 'blocker']),
               default='list',
               help='Mode to use to find bugs')
 @click.option("--status", 'status',
               multiple=True,
+              default=None,
               required=False,
-              default=['MODIFIED', 'VERIFIED', 'ON_QA'],
               type=click.Choice(constants.VALID_BUG_STATES),
-              help="Status of the bugs")
+              help="Include bugs of this status")
+@click.option("--exclude-status", 'exclude_status',
+              multiple=True,
+              default=None,
+              required=False,
+              type=click.Choice(constants.VALID_BUG_STATES),
+              help="Exclude bugs of this status. Useful when using default statuses")
 @click.option("--id", metavar='BUGID', default=None,
               multiple=True, required=False,
               help="Bugzilla IDs to add, required for LIST mode.")
 @click.option("--cve-trackers",
               required=False,
+              default=None,
```

Comment:
I wasn't sure if there was a way to set this to False if the default was True. Like `--cve-trackers=False`, if not the case I'll revert this change. Part of the exploration of "different modes have different defaults, but if a user explicitly sets it then use that".

Reply:
i'm pretty sure `--cve-trackers=False` works fine